numpy/add_newdocs.py

```diff
@@ -3096,17 +3096,11 @@ def luf(lamdaexpr, *args, **kwargs):
 
 
 add_newdoc('numpy.core.multiarray', 'ndarray', ('__copy__',
-    """a.__copy__([order])
+    """a.__copy__()
 
-    Return a copy of the array.
-
-    Parameters
-    ----------
-    order : {'C', 'F', 'A'}, optional
-        If order is 'C' (False) then the result is contiguous (default).
-        If order is 'Fortran' (True) then the result has fortran order.
-        If order is 'Any' (None) then the result has fortran order
-        only if the array already is in fortran order.
+    Used if copy.copy is called on an array. Returns a copy of the array.
```

Comment:
Nit: should be possible to intersphinx link to the python docs, maybe as simple as adding backticks.

Reply:
Should I also change this with `__deepcopy__` and the link to `copy.deepcopy`?